etc/docker/tests/run_test.sh

```diff
@@ -1,17 +1,8 @@
 #!/bin/bash
 set -e
 
-mkdir build
-cd build
-cmake .. -G Ninja -DVIAMCPPSDK_USE_DYNAMIC_PROTOS=ON \
-		-DVIAMCPPSDK_OFFLINE_PROTO_GENERATION=ON \
-		-DVIAMCPPSDK_SANITIZED_BUILD=$BUILD_SHARED \
-		-DVIAMCPPSDK_CLANG_TIDY=ON \
-		-DBUILD_SHARED_LIBS=$BUILD_SHARED
-
-ninja all
-ninja install
 INSTALL_DIR="$(pwd)/install"
+BUILD_SHARED=$(grep 'BUILD_SHARED_LIBS' CMakeCache.txt | cut -d '=' -f 2)
```

Comment:
are we adding this line because of the other shared library changes in #309 that are happening?

Reply:
no not super related except to the extent that this is part of the ongoing saga of having shared/static builds properly supported :) but in particular this script is building with CMake rather than Conan so it already works properly